docs/tasks/retiring-a-smart-answer.md

```diff
@@ -35,26 +35,25 @@ bundle exec rake "publishing_api:unpublish_redirect[d0a95767-f6ab-432a-aebc-096e
 
 If there is not content to replace the Smart Answer then the convention is to
 serve a page that indicates there used to be content but isn't anymore (a 410
-gone HTTP response). An example of how to perform this is:
+gone HTTP response). To do this you'd run the following command:
 
 ```
 bundle exec rake publishing_api:unpublish_gone[<content_id>]
 ```
 
-> It is possible to remove the Smart Answer and rather than serve a 410 Gone
-> page instead serve a 404 Not Found, which misleads visitors to the page that
-> content had never been published there. This can be done by using the
-> `publishing_api:unpublish_vanish` task instead of the gone equivalent. This
-> task should only be used in exceptional circumstances for example an
-> accidental early publishing of content with a sensitive URL.
+> It is possible to remove the Smart Answer and serve a 404 Not Found response (instead of a
+> 410 Gone response) which misleading implies that content had never been published there.
+> This can be done by using the `publishing_api:unpublish_vanish` task instead of the gone
```

Comment:
recognising this isn't new - maybe we can put in some subheadings and link back... 'gone equivalent' is a weird way to say it 😓